.github/workflows/extensive.yml

```diff
@@ -122,6 +122,9 @@ jobs:
         run: |
           echo $GITHUB_WORKSPACE"/swig/swigwin-3.0.12/" >> $GITHUB_PATH
           echo RUNTIME_LINKING_OPTION="-DWITH_STATIC_RUNTIME=ON" >> "${GITHUB_ENV}"
+          ./dev/utilities/expdef/expdef64.exe -dRlib.def -l R.dll
```

Comment:
I'm surprised this found the R.dll, i would have expected to need the full path to the R runtime.

Reply:
[This quick test](https://github.com/alessandrofelder/libsbml/runs/2964275078?check_suite_focus=true) suggests that the DLL is on the PATH of the GHA Windows runner: `which R.dll` evaluates correctly to `/c/Program Files/R/R-4.1.0/bin/x64/R.dll`.